docs/client_advanced.rst

```diff
@@ -56,6 +56,12 @@ For *text/plain* ::
 
     await session.post(url, data='Привет, Мир!')
 
+.. note::
+   ``Authorization`` header will be removed if you get redirected
+   to a different host or protocol, except the case when  ``HTTP -> HTTPS``
+   redirect is performed on the same host.
```

Comment:
Plz keep that versionchanged with a very small note.

Reply:
I added `versionchanged` with text from `CHANGES/5783.feature`